### ISSUE TITLE
Heads up, ActiveAdmin v4 is coming!

### ISSUE THREAD

**Issue by jgmontoya** [platanus/activeadmin_addons#500]
Hello plataneros, just letting you know that ActiveAdmin [V4 is somewhat near](https://github.com/activeadmin/activeadmin/releases?q=v4.0&expanded=true) and it looks awesome.

I have no idea when it'll be out of beta, if addons is still maintained (please confirm @rjherrera) or what the level of effort for supporting it would be.

[Here](https://github.com/activeadmin/activeadmin/blob/master/UPGRADING.md#breaking-changes) is an explainer of the breaking changes.

Pinging @difernandez, @gmq and @ldlsegovia just in case I might tickle their interest in this 😅 😄

**Comment by rjherrera** [comment]
Hi there 🙌🏻 ! I'm really excited about AA v4 too haha. 

I haven't looked much into it to understand whether the update requires too much effort or not (regarding AAA), but at least in one project we have, we will push to update AA to v4. So, as we rely heavily on AAA, we will most likely be compelled to upgrade this gem too.

That being said, I don't really think we will start looking into it before there is a non-beta release or at least a release date for v4 😅. 

Thanks for the heads up! I heard about v4 in January and had almost forgotten about it since then.

**Comment by matgaw** [comment]
@difernandez do you plan to continue maintaining this gem? it seems that the activity abruptly stopped a year ago

**Comment by difernandez** [comment]
@matgaw no, sorry (and sorry @jgmontoya 😅), I don't plan to, I no longer work at the company maintaining it

**Comment by matgaw** [comment]
@difernandez is there anyone else who was access and is willing to continue maintaining this gem?

**Comment by thisismydesign** [comment]
I installed `activeadmin_addons` on a project with v4 and I get `uninitialized constant ActiveAdmin::Views::Pages (NameError)` on startup.

**Comment by vnazarenko** [comment]
any news or plans about migrating to AA 4?

**Comment by griffero** [comment]
Any update on this? It'd be sad to stop using this gem.

**Comment by thisismydesign** [comment]
I ended up using `importmaps` (the default of AA v4) and `stimulus` (default of Rails since v7) to integrate JS libraries into AA v4, for example these show you how to add [tom-select](https://tom-select.js.org/):
- https://stackoverflow.com/questions/78985376/activeadmin-v4-how-to-add-custom-js-via-importmaps
- https://stackoverflow.com/a/79352359/2771889

Based on this, someone could make an update to `activeadmin_addons`, or create a similar gem for AA v4, but I'm not sure if it's needed at all. You can just integrate JS explicitly where you need it.
